exercise1/hash_1.py

```diff
@@ -12,8 +12,8 @@ class Transaction:
 
     def hash(self) -> bytes:
         """
-        TODO -- zaimplementuj funckję hashującą transakcje wykorzystując funkcję hash z simple_cryptography
-        wykorzystaj metodę int.to_bytes(2, 'big') oraz bytes(string, 'utf-8') do konwersji int to bytes
-        bytes mozna konkatenować
+        TODO: zaimplementuj funkcję hashującą transakcje, wykorzystując funkcję hash z simple_cryptography.
+            Wykorzystaj metodę int.to_bytes(2, 'big') oraz bytes(string, 'utf-8') do konwersji int to bytes.
+            Bytesz można konkatenować.
```

Comment:
I've added indent to comments because this way it works better with pycharm/intellij.
Before: 
<img width="854" alt="image" src="https://user-images.githubusercontent.com/52135326/231768086-04d1c8e3-7b02-4ca4-9834-12080bf356b2.png">

After:
<img width="874" alt="image" src="https://user-images.githubusercontent.com/52135326/231768027-f233d236-b38b-4a69-a9f1-4813fb09907c.png">

Reply:
`Bytesz`?